python/hello-world/e2etest.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
Please consider using nox/tox and py.test. You'll have to at minimum use py.test in order for this to be merged into python-docs-samples.